.github/workflows/ci.yml

```diff
@@ -274,7 +269,7 @@ jobs:
         if: steps.ninja-build-cache.outputs.cache-hit != 'true'
         uses: actions/setup-python@v5
         with:
-          python-version: '3.10'
+          python-version: "3.10"
```

Comment:
Could you do the reformatting in a separate commit?

Reply:
Oops, of course.